C++/day15.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+#include <cstddef>
+using namespace std;	
+class Node
+{
+    public:
+        int data;
+        Node *next;
+        Node(int d){
+            data=d;
+            next=NULL;
+        }
+};
+class Solution{
+    public:
+    Node* insert(Node *head,int _data) {
+        Node* _new = new Node(_data);
+        _new->next = NULL;
+        if (head == NULL) {
+            // don't forget the initial case!
+            // head is NULL b.a.e. on first run
+            head = _new;
+        } else if (head->next == NULL) {
```

Comment:
If you want you can do it. Please rename also the solution to `day15.1.cpp` since another solution called `day15.cpp` already exists. When you commit the changes you don't need to re-issue a pull request since github will automatically update this one! :smile: